ros/src/util/packages/autoware_launcher/documents/plugin/create.md

```diff
@@ -0,0 +1,39 @@
+
+## Create Plugin File
+
+If you want to use your launch file. Create the plugin file. For example, you can add ndt matching as following.<br>
+
+1. Create plugin file template by text editor (e.g. autoware_launcher/plugins/refs/ndt_matching.yaml).
+```
+    format: Autoware Launcher Plugin Version 0.1
+    rosxml: $(find lidar_localizer)/launch/ndt_matching.launch
+```
+1. Launch plugin file generate tool.<br>
+`rosrun autoware_launcher tool refs/ndt_matching`
+1. Set type and default value.<br>
```

Comment:
```suggestion
1. This will read the launch file indicated in the plugin file, and create a window with all the contained parameters. On this, the correct type and default values can be set.<br>
```